CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Updates the table component to add a `With filter` option ([PR #2803](https://github.com/alphagov/govuk_publishing_components/pull/2803))
```

Comment:
CHANGELOG entries should match the title of the PR.

Reply:
Updated 👍